docs/manage/fleet/robots.md

```diff
@@ -20,51 +20,34 @@ Add a new robot by providing a name in the **New Robot** field and clicking **AD
 
 Click the name of a robot to go to that robot's page, where you'll find a variety of tools for working with your robot.
 
-### Delete a robot
-
-You can delete a robot by checking the **Sure?** box in the lower left of the robot page and clicking **DELETE ROBOT**.
-
-![The DELETE ROBOT button and the confirmation checkbox (Sure?) next to it.](../../img/app-usage/delete.png)
-
 ## Navigating the robot page
 
 The banner at the top of the robot page displays the robot's location, name, and a drop down list of all parts of that robot.
-The first part you create will be the _main part_ but you can create additional parts in the drop down.
-
-{{< figure src="../../img/app-usage/part-drop-down.png" width="800px" alt="The robot page for an example robot with the parts drop down open." title="Example Robot Page with the Parts drop-down open (boxed in red)." >}}
+The first part you create is the _main part_ but you can create additional parts in the drop down.
 
+![The robot page for an example robot with the parts drop down open.](../../img/app-usage/part-drop-down.png)
 To delete a part or make it the main part, use the buttons in the top right of the **CONFIG** tab.
 
-{{< figure src="../../img/app-usage/part-mgmt.png" width="800px" alt="Screenshot of the CONFIG tab of a robot's page noting the location of the Make main part and Delete Part buttons." title="Screenshot of the CONFIG tab of a robot's page highlighting the Make main part and Delete Part buttons (boxed in red)." >}}
+![The CONFIG tab of a robot's page noting the location of the Make main part and Delete Part buttons.](../../img/app-usage/part-mgmt.png)
 
-If you've connected your robot to a machine running `viam-server` (instructions below), the banner also displays when the robot was last online, which version of `viam-server` it is running, the host name, the IP address or addresses, and its operating system.
+If you've connected your robot to a machine running `viam-server`, the banner also displays when the robot was last online, which version of `viam-server` it is running, the host name, the IP address or addresses, and its operating system.
 
 The following tabs are found on the robot page:
 
 ### Setup
 
 The **SETUP** tab contains information for starting an instance of `viam-server` on your robot's computer.
-Be sure to select the correct **Mode** and **Architecture** for your system in the upper left of the tab.
+Select the correct **Mode** and **Architecture** for your system in the upper left of the tab.
```

Comment:
[minor] This edit makes it sound like the first step of a series of steps, which it isn’t.